js/Main.js

```diff
@@ -1,4 +1,24 @@
 let adventure_btn = document.getElementById("open_adventure_page")
+let HTML_account_name = document.getElementById("HTML_account_name")
+let HTML_account_money = document.getElementById("HTML_account_money")
+let HTML_account_gems = document.getElementById("HTML_account_gems")
+let HTML_account_level = document.getElementById("HTML_account_lvl")
+let HTML_level_progress_bar = document.getElementById("lvl_progress_bar")
+let HTML_account_exp_in_progress_bar = document.getElementById("HTML_account_exp")
+
+let retrievedObject = JSON.parse(localStorage.getItem("data"))
+let account_name_in_LS = retrievedObject.user_data.user_name
+let account_money_in_LS = retrievedObject.user_data.money
+let account_gems_in_LS = retrievedObject.user_data.gems
+let account_level_in_LS = retrievedObject.user_data.level
+let account_exp_in_LS = retrievedObject.user_data.experience
+
+HTML_account_name.outerText = account_name_in_LS
+HTML_account_money.outerText = account_money_in_LS
+HTML_account_gems.outerText = account_gems_in_LS
+HTML_account_level.outerText = `Level: ${account_level_in_LS}`
+HTML_account_exp_in_progress_bar.outerText = account_exp_in_LS
+HTML_level_progress_bar.style = `width: ${100 - ((account_level_in_LS * 100 * 1.6) - account_exp_in_LS) / (account_level_in_LS * 100 * 1.6) * 100}%`
```

Comment:
1) Think about how you can shorten this formula.
2) First you have to calculate the percentage in a separate line, and then pass the result to the object